Add versionless value for admin api and terraform (#5741)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.

> For the dbt Cloud Admin API and Terraform, use the value `versionless` to configure your environment. 

Author: not sure if I should specify that users can input the value in the dbt_version key ?

Reviewer: Yes, I think that's a good clarification - "To upgrade an environment via Admin API or Terraform, set `dbt_version` to the string `"versionless"`

> For the dbt Cloud Admin API and Terraform, use the value `versionless` to configure your environment. 

Author: ```suggestion
To upgrade an environment in the dbt Cloud Admin API or Terraform, set `dbt_version` to the string `versionless`.
```

> To upgrade an environment in the dbt Cloud Admin API or Terraform, set `dbt_version` to the string `versionless`.

Author: ```suggestion
To upgrade an environment in the [dbt Cloud Admin API](/docs/dbt-cloud-apis/admin-cloud-api) or [Terraform](https://registry.terraform.io/providers/dbt-labs/dbtcloud/latest), set `dbt_version` to the string `versionless`.
```
